docker/compose/validator-testnet/docker-compose.yaml

```diff
@@ -19,7 +19,7 @@ services:
     validator:
         # Note this image currently does not support this, will update to the appropriate minimum
         # version shortly
-        image: libra/validator:master_f53da21f
+        image: libra/validator:master_609c0b05
```

Comment:
maybe we should actually give a url where they can discover the latest version and we can poke the team about labels or rename this to testnet?

Reply:
This is already defined a few lines above; `# * Images can be found at https://hub.docker.com/r/diem/validator/tags, obtain the latest tag and update below.`